client/src/utils/ReviewStar.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function ReviewStar({ currentReviewsMetadata }) {
+  function calAvgRating(MetadataRating) {
+    let total = 0;
+    let number = 0;
+    for (let i = 0; i < 5; i++) {
```

Comment:
Should we just have `i` go between 1 and 5, instead of 0 and 4 and adding 1 every time?